host-vm/install.sh

```diff
@@ -40,6 +40,7 @@ EOF
 #!/bin/bash
 $QEMU -name $VMNAME -M q35 -accel kvm -cpu host -m 4G -smp 4 $QARGS \
 -uuid $HOST_SYS_UUID \
+-debugcon file:bootlog -global isa-debugcon.iobase=0x402 \
```

Comment:
@mrabeda here's the log file I generated with this version of the ./start.sh script.  This file contains a cold boot followed by a warm "reboot".